obspy/core/inventory/response.py

```diff
@@ -1620,16 +1621,22 @@ def get_evalresp_response_for_frequencies(
         :type end_stage: int, optional
         :param end_stage: Stage sequence number of last stage that will be
             used (disregarding all later stages).
+        :type hide_sensitivity_mismatch_warning: bool
+        :param hide_sensitivity_mismatch_warning: Hide the evalresp warning
+            that computed and reported sensitivities don't match.
```

Comment:
```suggestion
            that computed and reported sensitivities do not match.
```

Reply:
Note that I copied the comment from legacy code so the "don't" is still around